router/src/validation.rs

```diff
@@ -4,12 +4,11 @@ use axum::http::StatusCode;
 use axum::Json;
 use thiserror::Error;
 use tokenizers::tokenizer::Tokenizer;
-use tokenizers::{
-    DecoderWrapper, ModelWrapper, NormalizerWrapper, PostProcessorWrapper, PreTokenizerWrapper,
-    TokenizerImpl,
-};
 use tokio::sync::{mpsc, oneshot};
 
+const MAX_MAX_NEW_TOKENS: u32 = 512;
```

Comment:
Why u32?

Reply:
@OlivierDehaene because it's compared with `request.parameters.max_new_tokens` which is u32. Could change it to `usize` and move where the cast is done, not sure it makes much difference?

Reply:
I was just wondering because you use `usize` in `MaxNewTokens(usize)`. I guess you could use `u32` everywhere.

Reply:
@OlivierDehaene I've now changed it to `usize`, but I just realized what you mean w.r.t. changing the enum parameter to `u32`. I guess I thought it looked weird for those enum parameter types to be inconsistent but it's probably better, will update again.